src/main/resources/application.yml

```diff
@@ -196,7 +196,7 @@ app:
 management:
   health:
     redis.enabled: ${REDIS_CACHE_ENABLED:false}
-    mongo.enabled: ${HEALTH_MONGO_ENABLED:true}
```

Comment:
Instead of setting the default value to false in order to disable mongo's health check, what about setting `HEALTH_MONGO_ENABLED: "false"` on helm values?

Later, you can re-enable it more easier